### ISSUE TITLE
BKB / Chronosphere

### ISSUE THREAD

**Issue by Azzurite** [kixelated/mechquiz#2]
Usually the interaction with chronosphere is if the ability does not work anymore in Chronosphere. For example Chrono / Evasion, chrono disables Evasion. Chrono / Omnislash, chrono does not stop omnislash. So there are two ways to think about BKB and Chrono as well: "Does BKB remove the effect of Chrono?" or "Does Chrono remove the effect of BKB?". The answer sought for in this "matchup" is if BKB negates the effects of chrono. But having BKB on will still block spells like it always does. So one could say BKB wins.

**Comment by sarmad3000** [comment]
I don't agree with this one, direct competition.. does BKB block effect of Chrono, answer is no so Chrono wins in my opinion.

**Comment by kixelated** [comment]
My intention was the simplest interpretation: does BKB allow you to move in chronosphere? I definitely see why people would get confused and I'll try to figure out a better way to word it.
